src/app/pages/playground/shared/compiler-selector-popup/compiler-selector-popup.component.html

```diff
@@ -26,7 +26,9 @@ <h1>Choose a SYCL Compiler</h1>
             @if (!compiler.enabled) {
               <div class="coming-soon">Coming Soon!</div>
             }
-            <img [src]="compiler.logo" alt="{{ compiler.name }} Logo" />
+            <div class="logo-container">
+              <img [ngSrc]="compiler.logo" alt="{{ compiler.name }} Logo" fill />
```

Comment:
Is this `fill` expected here?

Reply:
That is a good spot!

But yeah, its a valid tag in the Angular world. It basically hints to the NgOptimizedImage functionality to fill the parent container.

https://angular.dev/guide/image-optimization#using-fill-mode